src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js

```diff
@@ -55,12 +55,15 @@ setTimeout(function () {
     }
   }
 
-  function updateCssByPath(path) {
+  async function updateCssByPath(path) {
     const styleElement = document.querySelector(`link[href^="${path}"]`) ||
       document.querySelector(`link[href^="${document.baseURI}${path}"]`);
 
+    // Receive a Clear-site-data header.
+    await fetch('/_framework/clear-browser-cache');
```

Comment:
Scoped css files can contain links to other css files. We'll try clearing out the http caches to force the browser to re-download.